src/app/screens/Home/index.js

```diff
@@ -26,41 +31,141 @@ const Home = ({
   dispatch,
   lastBill,
   lastBillError,
-  lastBillloading
+  lastBillloading,
+  digitalBilling,
+  digitalBillingLoading
 }) => {
   useEffect(() => {
     if (isEmpty(accounts) && !accountsError && !accountsLoading) dispatch(AccountActions.getAccounts());
     if (objectIsEmpty(lastBill) && !lastBillError && !lastBillloading) dispatch(BillsActions.getLastBill());
   }, []);
 
+  useEffect(() => {
+    if (!objectIsEmpty(currentAccount)) {
+      dispatch(
+        BillsActions.setDigitalBilling({
+          adherido_factura_digital: currentAccount.adherido_factura_digital,
+          contact_emails: currentAccount.contact_emails
+        })
+      );
+    }
+  }, [currentAccount]);
+
+  const [tagStatus, setTagStatus] = useState({});
+
+  useEffect(() => {
+    if (digitalBilling.adherido_factura_digital) {
+      setTagStatus(tagState.success);
+    } else {
+      setTagStatus(tagState.error);
+    }
+  }, [digitalBilling]);
+
+  const [inputValue, setInputValue] = useState('');
+  const [modalStatus, setModalStatus] = useState(modalState.closed);
+
+  const handleCta = action => {
+    if (action === 'modify') {
+      dispatch(BillsActions.digitalBillingUpdate(inputValue));
+      setModalStatus(modalState.closed);
+    }
+
+    if (action === 'subscribe') {
+      dispatch(BillsActions.digitalBillingSubscription(inputValue));
+      setModalStatus(modalState.closed);
+    }
+
+    if (action === 'unsubscribe') {
+      dispatch(BillsActions.digitalBillingUnsubscription(inputValue));
+      setModalStatus(modalState.closed);
+    }
+  };
```

Comment:
```suggestion
  const handleCta = action => {
    if (action === 'modify') dispatch(BillsActions.digitalBillingUpdate(inputValue));
    if (action === 'subscribe') dispatch(BillsActions.digitalBillingSubscription(inputValue));
    if (action === 'unsubscribe') dispatch(BillsActions.digitalBillingUnsubscription(inputValue));
    setModalStatus(modalState.closed);
  };
```
O bien podría ser un switch también!